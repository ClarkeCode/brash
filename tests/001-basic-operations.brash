print true
print false

print "Logical AND"
print true  && true
print true  && false
print false && true
print false && false

print "Logical OR"
print true  || true
print true  || false
print false || true
print false || false

print "Logical XOR"
print true  ^^ true
print true  ^^ false
print false ^^ true
print false ^^ false

print "Arithmetic Operations"
print 1
print 1.22
print -1
print 1 + 1
print 3 - 2
print 1.5 * 2
print 7 / 3
print 3.3 % 2

print "Strings"
print "Hello"
print "Bugs" + " " + "Bunny"

