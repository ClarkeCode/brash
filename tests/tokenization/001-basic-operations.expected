true
false
Logical AND
true
false
false
false
Logical OR
true
true
true
false
Logical XOR
false
true
true
false
Arithmetic Operations
1.000000
1.220000
-1.000000
2.000000
1.000000
3.000000
2.333333
1.300000
Strings
Hello
Bugs Bunny
